src/projectscene/view/clipsview/selectionviewcontroller.cpp

```diff
@@ -33,11 +80,23 @@ void SelectionViewController::onPositionChanged(double x, double y)
         return;
     }
 
+    Qt::KeyboardModifiers modifiers = QApplication::keyboardModifiers();
+
+    //! NOTE: always treat simultaneously pressed Ctrl and Shift as Ctrl
```

Comment:
it looks like we can move it to a separate function and reuse

Reply:
fixed 👍